app/src/main/java/com/skyyo/samples/application/activity/MainActivity.kt

```diff
@@ -89,12 +89,8 @@ class MainActivity : AppCompatActivity() {
                 val callback = NavController.OnDestinationChangedListener { _, destination, _ ->
                     when (destination.route) {
                         Destination.SampleContainer.route -> {
-                            clearUserIdlingSession()
                         }
                         else -> {
-                            if (destination.route != Destination.UserInteractionTrackingResult.route) {
-                                userIdlingSessionEventDispatcher.startSession()
-                            }
                         }
```

Comment:
looks like needless construction

Reply:
What construction? It looks like its already removed

Reply:
```
when (destination.route) {
   Destination.SampleContainer.route -> {
 }
   else -> {
  }
 }
```
this one

Reply:
This construction was here before me, I think there's no need to remove it

Reply:
okay) but seems it doing nothing

Reply:
It's here just for example